src/components/helpers.js

```diff
@@ -155,6 +155,9 @@ export const normalizeTransferHistoryMessage = (transferData, sender) => {
 		},
 		department: () => {
 			const to = nextDepartment && nextDepartment.name;
+			if (!sender.username) {
+				return I18n.t('the agent transferred the chat to the department %{to}', { to });
```

Comment:
```suggestion
				return I18n.t('The agent transferred the chat to the department %{to}', { to });
```

A bit nitpicky but 🙈

Reply:
Wont this conflict with the key? `the_agent_transferred_the_chat_to_the_department_t_3cbe495c` seems to not have caps

Reply:
changes done